src/test/java/picard/illumina/IlluminaLaneMetricsCollectorTest.java

```diff
@@ -77,4 +83,54 @@ public void testCollectIlluminaLaneMetrics(final String testRun, final ReadStruc
                 {"A67HY", new ReadStructure("8B8B")}
         };
     }
+
+    /** Ensures that an exception is thrown when we encounter a tile without phasing/pre-phasing metrics. */
+    @Test(expectedExceptions = PicardException.class)
```

Comment:
Can you add a test that is only missing phasing _**or**_ prephasing and see that even in SILENT mode it blows up? (I don't know how hard it is to cook up the data like that...)